solutions/remotemonitoring/single-vm/docker-compose.java.yml

```diff
@@ -99,7 +99,12 @@ services:
       - PCS_AUTH_REQUIRED
       - PCS_CORS_WHITELIST
       - PCS_APPLICATION_SECRET
+      - PCS_AAD_TENANT
       - APPLICATION_SECRET
```

Comment:
nit: keep these in the same order as solutions/remotemonitoring/single-vm/docker-compose.dotnet.yml